tests/unit/components/graphqlFormGenerator/editRuntimeForm.vue.spec.js

```diff
@@ -96,7 +108,8 @@ describe('EditRuntimeForm Component', () => {
     it("queries the task's runtime section & processes the response", async () => {
       const wrapper = mountFunction({
         props,
-        created () {}
+        created () {},
+        propsData: data,
```

Comment:
`propsData` is obsolete. Make sure you are looking at the correct version of the docs: https://test-utils.vuejs.org/api/#data